toolkit/tools/imagecustomizerapi/disksize.go

```diff
@@ -72,8 +91,8 @@ func parseDiskSize(diskSizeString string) (DiskSize, error) {
 	}
 
 	// The imager's diskutils works in MiB. So, restrict disk and partition sizes to multiples of 1 MiB.
-	if num%diskutils.MiB != 0 {
-		return 0, fmt.Errorf("(%d) must be a multiple of 1 MiB", num)
+	if num%DefaultPartitionAlignment != 0 {
+		return 0, fmt.Errorf("(%s) must be a multiple of 1 MiB", diskSizeString)
```

Comment:
I'd use the `DefaultPartitionAlignment` constant in the error message as well, so that it if the value ever changes, the error will also automatically update.

Reply:
Good call.